app/src/main/res/layout/context_menu.xml

```diff
@@ -12,16 +11,20 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_margin="0dp"
-        android:padding="0dp"
-        android:background="@android:color/transparent" />
+        android:background="@android:color/transparent"
+        android:paddingTop="0dp"
+        android:paddingStart="8dp"
```

Comment:
Our hit targets look like this now:

![screenshot_1507940598](https://user-images.githubusercontent.com/759372/31570710-96001552-b03b-11e7-9ab6-43990d911c33.png)

I assume we want the hit targets to span the full width of the view: would it be possible to fix this? Perhaps we can add the padding to the NavigationView children instead.

Reply:
Ok. Will be handle in the next commit